src/app/views/sessions/signin/signin.component.ts

```diff
@@ -260,7 +263,9 @@ export class SigninComponent implements OnInit {
   }
 
   onGoToLegacy() {
-    this.dialogService.confirm(T("Log in to Legacy User Interface?"), "", true, T('Continue')).subscribe((res) => {
+    this.dialogService.confirm(T("Warning:"), 
```

Comment:
```suggestion
    this.dialogService.confirm(T("Warning"),
```
The `:` appears to be unnecessary for this dialog.